app/models/family.rb

```diff
@@ -86,6 +88,10 @@ def everyone_has_birthdates?
   def anyone_has_email?
     attendees.any? { |p| p.email.present? }
   end
+ 
+  def anyone_registered_for_legacy_conferences()
+    attendees.find{|a| a.conferences.find{ |c| c.name == 'Legacy'}} != nil
```

Comment:
Same comments as other PR

Reply:
Yah, i required some of the previous changes when I was working on it, so it was built based of the other Branch.